filebeat/crawler/prospector_log.go

```diff
@@ -31,7 +29,7 @@ func NewProspectorLog(p *Prospector) (*ProspectorLog, error) {
 	}
 
 	// Init File Stat list
-	prospectorer.harvesterStats = make(map[string]input.FileStat)
+	prospectorer.harvesterStates = make(map[string]input.FileState)
```

Comment:
Make removed. You know how much fan I am of full variables names vs short names ;-)